### ISSUE TITLE
[BUG] - PF2e animations play on missed hit even when option disabled

### ISSUE THREAD

**Issue by blacktrench** [otigon/automated-jb2a-animations#576]
I'm on foundry 10 - build 291

I disabled all modules except auto animations, jb2a, sequence, and socketlib.

When the "Play animations on Hits and Misses is disabled they still go off see image below for an unarmed strike (magical green etc.) with feature disabled.

![image](https://user-images.githubusercontent.com/24504912/213984178-ac6d778e-12dd-46df-af90-81b43a63824b.png)

Hopefully this is helpful - cheers! :)

**Comment by otigon** [comment]
As far as I've tested, this option is working as intended. When disabled, it always hits on the token, and when enabled will hit a spot around the token when missed

**Comment by otigon** [comment]
If you can take a quick video perhaps I can see better what is happening

**Comment by blacktrench** [comment]
Ah, looks like I miscommunicated - sorry - very little sleep when submitting it. My issue is with the fact that animations should only play on damage rolls (but as I usually have it set to auto do damage rolls on hits I was looking at that as the issue)

I did some further testing and PCs it works flawlessly but for NPCs the animations trigger on attack rolls vs. the damage rolls

Video below with all other mods disabled showing pc vs. npc

https://user-images.githubusercontent.com/24504912/214370916-778a8fde-a6af-43f2-b24d-659f68384b44.mp4
